### PR TITLE
Removed extra space between block name and block state

### PR DESCRIPTION
**Pull request by IceTank**

### CODE REVIEW

**Review by u9g — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -135,7 +135,7 @@ class Schematic {
       let state
       if (versionedMcData.isNewerOrEqualTo('1.13')) {
         state = Object.entries(block.getProperties()).map(([key, value]) => `${key}="${value}"`).join(',')
-        state = state ? ` [${state}]` : ''
```
Comment (SeanMcCord) [inline index.js:138]: I did some digging and found that the command differs between minecraft editions. Specifically MCPE(bedrock) includes the space while java does not[1]. A simple fix may be to switch on [`versionedMcData.type`](https://github.com/PrismarineJS/node-minecraft-data/blob/master/doc/api.md#type)

[1] https://minecraft.fandom.com/wiki/Commands/setblock#Examples

**Review by SeanMcCord — CHANGES_REQUESTED** [review]
Minor change to allow for compatibility between java and bedrock.

File: index.js (hunk 1 above)
Comment (IceTank) [reply index.js:138]: Interesting. I will change the pull request to account for that.

File: index.js [hunk 2]
```diff
@@ -127,15 +127,19 @@ class Schematic {
...
       const { x, y, z } = pos.offset(offset.x, offset.y, offset.z)
       const versionedMcData = mcData(this.version)
       let state
       if (versionedMcData.isNewerOrEqualTo('1.13')) {
         state = Object.entries(block.getProperties()).map(([key, value]) => `${key}="${value}"`).join(',')
-        state = state ? `[${state}]` : ''
+        if (platform === 'pc') {
+          state = state ? `[${state}]` : ''
+        } else {
+          state = state ? ` [${state}]` : ''
```
Comment (SeanMcCord) [inline index.js:141]: Nit: I'd suggest explicitly checking if the platform is `'pe'` here and throw an error otherwise.

**Review by SeanMcCord — APPROVED** [review]
LGTM: one trivial nit.

File: index.js (hunk 2 above)
Comment (IceTank) [reply index.js]: Fixed

**Review by rom1504 — APPROVED** [review]

**Review by rom1504 — APPROVED** [review]

**Pull request merged**